src/services/teams/index.js

```diff
@@ -314,21 +310,33 @@ class Add {
 		}
 		return Promise.all([
 			// eslint-disable-next-line no-underscore-dangle
-			this._generateLink({
-				esid, email, teamId, importHash,
-			}, isUserCreated),
+			this._generateLink(
+				{
+					esid,
+					email,
+					teamId,
+					importHash,
+				},
+				isUserCreated
+			),
 			patchTeam(this, teamId, { invitedUserIds }, params),
 			// eslint-disable-next-line no-underscore-dangle
-		]).then(([linkData]) => Add._response({
-			linkData, user, isUserCreated, isResend, email,
-		}));
+		]).then(([linkData]) =>
+			Add._response({
```

Comment:
Codacy found an issue: [Unexpected dangling '_' in '_response'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)